tests/test_transformers/test_timeseries_transformer.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+from sklego.preprocessing import DateTimeFeatures
+import pytest
+
+
+@pytest.fixture()
+def timeseries_df():
+    # Generate example dataset
+    X = pd.DataFrame({
+        "date": pd.to_datetime(["2017-1-19", "2017-12-25", "2018-4-1", "2018-8-8 21:00",
+                                "2019-4-21 12:15:00"]),
+        "amount": [85, 61, 38, 74, 18],
+    })
+    y = [0, 1, 1, 0, 0]
+    return X, y
+
+
+def test_timeseries_feature_adder(timeseries_df):
+    """
+    Test that the DateTimeFeatures function works as expected.
+    :param timeseries_df: produces the samples data frame
```

Comment:
for tests you don't need to set a docstring per se since it won't be documented for the end user. adding the notification as a comment is much appreciated though!